recipe/build.sh

```diff
@@ -110,6 +110,72 @@ export TF_CONFIGURE_IOS=0
 sed -i -e "/PROTOBUF_INCLUDE_PATH/c\ " .bazelrc
 sed -i -e "/PREFIX/c\ " .bazelrc
 
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    export GCC_HOST_COMPILER_PATH="${GCC}"
+    export GCC_HOST_COMPILER_PREFIX="$(dirname ${GCC})"
+    # export CFLAGS=$(echo $CFLAGS | sed 's:-I/usr/local/cuda/include::g')
+    # export CPPFLAGS=$(echo $CPPFLAGS | sed 's:-I/usr/local/cuda:-isystem/usr/local/cuda:g')
+    # export CXXFLAGS=$(echo $CXXFLAGS | sed 's:-I/usr/local/cuda:-isystem/usr/local/cuda:g')
+    export TF_CUDA_PATHS="${PREFIX},/usr/local/cuda-${cuda_compiler_version},/usr"
+    export CLANG_CUDA_COMPILER_PATH=${PREFIX}/bin/clang
+    export USE_CUDA=1
+    export cuda=Y
+    export TF_NEED_CUDA=1
+    export TF_CUDA_VERSION="${cuda_compiler_version}"
+    export TF_CUDNN_VERSION="${cudnn}"
+    export TF_NCCL_VERSION=$(pkg-config nccl --modversion | grep -Po '\d+\.\d+')
+    export NCCL_ROOT_DIR=$PREFIX
+    export USE_STATIC_NCCL=0
+    export USE_STATIC_CUDNN=0
+    export PATH="${CUDA_HOME}/bin:$PATH"
+    export CUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME
+    export LDFLAGS="${LDFLAGS//-Wl,-z,now/-Wl,-z,lazy}"
+    export CC_OPT_FLAGS="-march=nocona -mtune=haswell"
+
+    if [[ ${cuda_compiler_version} == 10.* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=5.2,5.3,6.0,6.1,6.2,7.0,7.2,7.5
+    elif [[ ${cuda_compiler_version} == 11.0* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=5.2,5.3,6.0,6.1,6.2,7.0,7.2,7.5,8.0
+    elif [[ ${cuda_compiler_version} == 11.1 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=5.2,5.3,6.0,6.1,6.2,7.0,7.2,7.5,8.0,8.6
+    elif [[ ${cuda_compiler_version} == 11.2 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=5.2,5.3,6.0,6.1,6.2,7.0,7.2,7.5,8.0,8.6
+    else
+        echo "unsupported cuda version."
+        exit 1
+    fi
+
+    # cuda builds don't work with custom_toolchain, instead we hard-code arguments, mostly copied
+    # from https://github.com/AnacondaRecipes/tensorflow_recipes/tree/master/tensorflow-base-gpu
```

Comment:
This is part of the approach that is likely to be rejected based on the discussion in https://github.com/conda-forge/tensorflow-feedstock/issues/134

Note also that if you just want a working cuda version for Linux I have that already in https://github.com/izahn/tensorflow-feedstock/tree/cuda_2.6.0 , https://github.com/izahn/tensorflow-feedstock/tree/cuda_2.5.1, and https://github.com/izahn/tensorflow-feedstock/tree/cuda_2.4.3 . Python 39 packages are available at https://anaconda.org/izahn/repo. It looks like these won't make it into conda-forge for the reasons discussed in https://github.com/conda-forge/tensorflow-feedstock/issues/134 but they work fine for me.